metagpt/actions/design_api.py

```diff
@@ -58,15 +58,15 @@
 """,
         "FORMAT_EXAMPLE": """
 [CONTENT]
-{
+{{
     "Implementation approach": "We will ...",
-    "project_name": "snake_game",
+    "Project name": "{project_name}",
     "File list": ["main.py"],
     "Data structures and interfaces": '
     classDiagram
-        class Game{
```

Comment:
This original format follows UML, I dont think `{{` makes it better for this prompt

Reply:
`{{` is used in string formatting, and the result returned by the format operation is `{`.